gems/activerecord/6.1/activerecord-6.1.rbs

```diff
@@ -5,4 +5,8 @@ module ActiveRecord
     end
     extend DelegatedType
   end
+
+  module Enum
+    def enum: (Hash[Symbol, (Hash[(Symbol | String), untyped] | Array[Symbol | String] | bool)] definitions) -> void
```

Comment:
ActiveRecord::Enum#enum method in 6.1 version
https://github.com/rails/rails/blob/f09dc7c4c2e8b9375345d443c230cb8d78ad6a18/activerecord/lib/active_record/enum.rb#L158